packages/devtools_app/lib/src/network/network_request_inspector_views.dart

```diff
@@ -151,6 +160,86 @@ class HttpResponseView extends StatelessWidget {
   }
 }
 
+class ImageResponseView extends StatelessWidget {
+  const ImageResponseView(this.data);
+
+  final DartIOHttpRequestData data;
+
+  @override
+  Widget build(BuildContext context) {
+    final img = image.decodeImage(data.encodedResponse);
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        _buildTile(
+          'Image Preview',
+          [
+            Padding(
+              padding: const EdgeInsets.all(
+                denseSpacing,
+              ),
+              child: Image.memory(
+                data.encodedResponse,
+              ),
+            ),
+          ],
+        ),
+        LayoutBuilder(builder: (context, constraints) {
```

Comment:
we aren't using the constraints anymore so we should be able to remove this layout builder